ql/src/test/queries/clientpositive/llapdecider.q

```diff
@@ -67,4 +67,10 @@ set hive.llap.skip.compile.udf.check=true;
 
 EXPLAIN SELECT sum(cast(test_udf_get_java_string(cast(key as string)) as int) + 1) from src_orc_n2 where cast(key as int) > 1;
 
-set hive.execution.mode=container;
\ No newline at end of file
+set hive.execution.mode=llap;
+set hive.llap.execution.mode=all;
+set hive.merge.tezfiles=true;
```

Comment:
Can you reset this back to the default value (false) in the end of this execution?

Rest LGTM.

Reply:
Will do @rbalamohan - thanks for taking a look. I have test failures I need to look at also - some compaction test related ones and various diff changes around things getting vectorized and llap converted that were not before. So I need some additional investigation.